kisee/kisee.py

```diff
@@ -73,7 +73,7 @@ def parse_args(program_args=None) -> argparse.Namespace:
         "-v",
         "--verbose",
         dest="loglevel",
-        default=0,
+        default=40,
```

Comment:
By setting --verbose to 40 by default you're setting a loglevel of -350 by default, instead of 50, why?
Currently default is loglevel=50 (critical) and every "-v" removes 10, so -v is ERROR, -vv is WARNING, -vvv is INFO, -vvvv is DEBUG.

Reply:
Right, I wanted 40 (ERROR) to be the final value of the logging. Went to fast haven't seen that we have function that take the value provided in default multiply it to conform to python logging values.

I changed it to 1, so let's have 40 (ERROR) by default ?